Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM golang:1.13.7-alpine as builder
 
 RUN apk add --no-cache gcc libc-dev git
 
-WORKDIR /src/litmus-service
+WORKDIR /src/keptn-service-template-go
```

Comment:
is this correct or should it be `src/litmus-service`?

Reply:
same would be true for the rest of the file

Reply:
Thanks, will take a look